compiler/luci/pass/src/FusePreActivationBatchNormPass.test.cpp

```diff
@@ -146,6 +156,8 @@ class SimpleGraph
   luci::CircleConst *pred_conv_filter;
   luci::CircleConst *pred_conv_bias;
   luci::CircleConv2D *pred_conv2;
+  luci::CircleConst *pred_conv2_filter;
```

Comment:
(maybe in another PR) for static code analysis tool warnings, would be better to initialize with nullptr
```
luci::CircleConst *pred_conv2_filter = nullptr;
```
for this and other members...